cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4136,4 +4136,12 @@
     </description>
   </property>
 
+  <property>
+    <name>system.program.scan.interval</name>
```

Comment:
Usually the unit is in the config name. E.g. `system.program.scan.interval.seconds`

Reply:
Done